units/meson.build

```diff
@@ -60,6 +60,11 @@ units = [
           'conditions' : ['ENABLE_INITRD'],
           'symlinks' : ['initrd.target.wants/'],
         },
+        {
+          'file' : 'systemd-bsod.service.in',
+          'conditions' : ['ENABLE_INITRD'],
+          'symlinks' : ['initrd.target.wants/'],
+        },
```

Comment:
Perhaps the 'symlinks'?

Reply:
no, the problem is the condition, it needs to also list `HAVE_QRENCODE` otherwise it will be installed when bsod is not